workbench/src/main/setupAddRemovePlugin.js

```diff
@@ -81,19 +81,19 @@ export function setupAddPlugin() {
         // Create a conda env containing the plugin and its dependencies
         const envName = `invest_plugin_${pluginID}`;
         await spawnWithLogging(
-          mamba,
+          micromamba,
           ['create', '--yes', '--name', envName, '-c', 'conda-forge', '"python<3.12"', '"gdal<3.6"']
         );
-        logger.info('created mamba env for plugin');
+        logger.info('created micromamba env for plugin');
         await spawnWithLogging(
-          mamba,
-          ['run', '--verbose', '--no-capture-output', '--name', envName, 'pip', 'install', `git+${pluginURL}`]
+          micromamba,
+          ['run', '--name', envName, 'pip', 'install', `git+${pluginURL}`]
         );
         logger.info('installed plugin into its env');
         // Write plugin metadata to the workbench's config.json
-        const envInfo = execSync(`${mamba} info --envs`, { windowsHide: true }).toString();
+        const envInfo = execSync(`${micromamba} info --name ${envName}`, { windowsHide: true }).toString();
         logger.info(`env info:\n${envInfo}`);
-        const regex = new RegExp(String.raw`^${envName} +(.+)$`, 'm');
+        const regex = /env location : (.+)/;
```

Comment:
I don't think I've seen this syntax before. Is the `/.../` syntax for a regex, without needing it to be wrapped in a string?

Reply:
Yes, it's a [regex literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_expressions#creating_a_regular_expression). My linter recommended this over a string.